src/py/_pyodide/_core_docs.py

```diff
@@ -527,9 +527,9 @@ class JsMutableMap(JsMap):
     number (idiomatically it should be called ``.size``) and it must be
     iterable.
 
-    Instances of the JavaScript builtin ``Map`` class are ``JsMutableMap``s.
+    Instances of the JavaScript builtin ``Map`` class are ``JsMutableMap`` s.
```

Comment:
I don't think we want these two spaces.

Reply:
It wasn't displayed correctly:

[before]

![image](https://user-images.githubusercontent.com/24893111/209739381-ebd664a7-48c7-46eb-a02b-5ca352182fde.png)

[after]

![image](https://user-images.githubusercontent.com/24893111/209739442-95f1e264-1e8b-4956-878f-185c0f96ab5b.png)